ray-operator/config/samples/ray-service.tpu-multi-host.yaml

```diff
@@ -0,0 +1,94 @@
+apiVersion: ray.io/v1
+kind: RayService
+metadata:
+  name: s-d-tpu
+spec:
+  serveConfigV2: |
+    applications:
+      - name: stable_diffusion
+        import_path: stable_diffusion.stable_diffusion_tpu:deployment
+        runtime_env:
+          working_dir: "https://github.com/ryanaoleary/serve_config_examples/archive/refs/heads/tpu-example.zip"
```

Comment:
As a side note, other samples in the kuberay repo have been putting source files within the same directory e.g.: https://github.com/ray-project/kuberay/blob/master/ray-operator/config/samples/pytorch-resnet-image-classifier/fine-tune-pytorch-resnet-image-classifier.py 

Wonder if we should do the same here instead of putting them in the serve_config_example repo in https://github.com/ray-project/serve_config_examples/pull/8

@kevin85421 what do you think? The benefit to putting it in the kuberay repo is that the whole example is contained in the same repo

Reply:
I'm fine with both options. The advantage of adding the YAML in the KubeRay repository is that we have complete control and can prevent unexpected updates to `serve_config_example`. However, the downside is that the KubeRay repository is much larger than `serve_config_example`, which could increase startup time.

Currently, the KubeRay repository is 187 MB. It increases the startup, but maybe not that much (depending on the user's network speed).

Reply:
I added the python code in `serve_config_example` since that's where the existing [stable diffusion example](https://github.com/ray-project/kuberay/blob/master/ray-operator/config/samples/ray-service.stable-diffusion.yaml) put it. I can move it if that's preferable, but `serve_config_example` doesn't seem to be changed much so I think it should be fine to leave it there.